### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kmb32123**
Created a contributing doc from the rules on [your web page](http://notepad-plus-plus.org/news/notepad-plus-plus-on-github.html).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#Contributing
+
+Your pull requests will be welcome; however, they may not be accepted for various reasons.
+
+##Guidelines for pull requests:
+
+1. Respect Notepad++ coding style.
+2. Make a single change per commit.
+3. Make your modification compact - don't reformat source code in your request. It makes code review more difficult.
+
```
Comment (ariccio) [inline CONTRIBUTING.md]: *"This pull request can be automatically merged by project collaborators"* (the golden message)

**Comment by ariccio** [comment]
:+1:  :clap:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#Contributing
+
+Your pull requests will be welcome; however, they may not be accepted for various reasons.
```
Comment (ariccio) [inline CONTRIBUTING.md]: > Your pull requests will be welcome

Your pull requests *are* welcome

**Comment by ariccio** [comment]
Maybe it should say something about testing changes [in Application Verifier](https://randomascii.wordpress.com/2011/12/07/increased-reliability-through-more-crashes/)?

**Comment by MAPJe71** [comment]
Where can we find a description of the coding style (hyperlink)?

**Comment by ariccio** [comment]
> Where can we find a description of the coding style (hyperlink)?

Huh...I can't actually find a link to it.

The best advice is probably: "follow the style of the code around you"

**Pull request merged**